src/main.cpp

```diff
@@ -142,6 +143,26 @@ void adjustScaleFactor(CmdlineArgs* pArgs) {
     }
 }
 
+void getStyleOverride(CmdlineArgs* pArgs) {
+    if (!pArgs->getStyle().isEmpty()) {
+        qDebug() << "Default style is overwritten by command line argument "
+                    "-style"
+                 << pArgs->getStyle();
+        QApplication::setStyle(pArgs->getStyle());
```

Comment:
Isn't this redundant?

Reply:
No it's not. Without the branch would be out of any effect.